cmd/fyne_demo/screens/container.go

```diff
@@ -0,0 +1,109 @@
+package screens
+
+import (
+	"fmt"
+	"image/color"
+
+	"fyne.io/fyne"
+	"fyne.io/fyne/canvas"
+	"fyne.io/fyne/layout"
+	"fyne.io/fyne/theme"
+	"fyne.io/fyne/widget"
+)
+
+// ContainerScreen loads a tab panel for containers and layouts
+func ContainerScreen() fyne.CanvasObject {
+	return widget.NewTabContainer(
+		widget.NewTabItem("Split", makeSplitTab()),
+		widget.NewTabItem("Scroll", makeScrollTab()),
+		// layouts
```

Comment:
If we had tab grouping (#899) we could surface this distinction

Reply:
Haha, that did occur to me - but without the context it would be meaningless.
Honestly I didn't like this a huge amount because it's not clear that some are layouts and that is an important distinction.
But when the layouts were all in a layout tab we had 3 tabs deep navigation which seemed very wrong.
We could put 2 tab controllers next to each other but that also feels bad.
Not sure that there is a perfect solution to this right now, maybe it needs more thought for layout?

Reply:
Hmm the deeper the navigation the less discoverable the content - definitely not what we want for a demo. Two tabbers next to each other feels bad to me too - there is no visual separation between the groups of tabs making it a confusing UI. You could use a splitter as that has a big divider, but again that doesn't feel right, maybe we should have Layouts and Containers as separate tabs?

Reply:
Too many tabs I think. I am happy enough with this as a compromise for now, this app will continue to evolve.